spec/system/smoke_test_spec.rb

```diff
@@ -2,14 +2,35 @@
 
 describe "DiscourseAutomation | smoke test", type: :system, js: true do
   fab!(:admin) { Fabricate(:admin) }
+  fab!(:group) { Fabricate(:group, name: "test") }
+  fab!(:badge) { Fabricate(:badge, name: "badge") }
 
   before do
-    Fabricate(:group, name: "test")
-    Fabricate(:badge, name: "badge")
     SiteSetting.discourse_automation_enabled = true
     sign_in(admin)
   end
 
+  it "allows default_value on fields" do
+    DiscourseAutomation::Scriptable.add("test") do
```

Comment:
It looks like this ends up defining a method but we are not cleaning stuff up at the end of the test. T

Reply:
yeah I don't think automation supports cleanup ... will add an undefine method somewhere.

Reply:
ok added @tgxworld

Reply:
Yes indeed good idea 👍